studio/schemas/npmDependency.js

```diff
@@ -0,0 +1,30 @@
+export default {
+  name: 'npmDependency',
+  title: 'npm Dependency',
```

Comment:
our "dependencies" are mostly npm libraries so this is specific, we could expand to other fields